blog.qmd

```diff
@@ -0,0 +1,329 @@
+
+Our analysis revealed that the medical and education sectors dominate WBL opportunities, which is expected given that these fields often require practical experience for licensure. Clinicals, largely offered in medical and related fields, and field experiences, predominantly in the education sector, were the most common types of WBL identified. However, we also found that apprenticeships, despite their value in career preparation and advancement ([Kuehn, Marotta, et al. 2022](https://www.urban.org/research/publication/beyond-productivity-how-employers-gain-more-apprenticeship)), are one of the least offered forms of WBL.
+
+It's important to note that our methodology has limitations. For instance, our current dataset lacks information on the quality of WBL experiences and dimensions of equity, such as whether the programs are paid. This is a significant consideration since many community college students work while enrolled in their courses and may not be able to devote time to an unpaid internship or other WBL program.
+
+Despite these limitations, this exploratory analysis provides a starting point for understanding the prevalence of WBL in community colleges and identifying opportunities for expansion. Future research could build on this work to develop more robust measures of WBL prevalence and to investigate the quality and equity of these opportunities.
+
+
+## Conclusion
+
+In this post, we've demonstrated how to use `quanteda` to analyze course
+descriptions and identify WBL opportunities in community
+colleges. While our analysis focused on Florida, the same methods could be
+applied to other states or regions. Extending the analysis to other states would provide a broader understanding of WBL opportunities in community colleges across the U.S., informing policy and identifying areas for expansion. This could help establish goals, support programs, and highlight best practices nationwide.
+
+`Quanteda` stands out for its ability to efficiently handle large volumes of text data and intuitive interface. It provides researchers with the flexibility to shape their analysis according to their specific needs, such as identifying key terms, comparing text documents, or investigating text patterns. Given that our study was exploratory in nature, this flexibility was particularly useful. It allowed us to quickly and efficiently examine novel text-data, making `quanteda` an excellent choice for text analysis. It is important to note that the functions used in this analysis only scratch the surface of the full functionality of `quanteda`. To learn more, you can read `quanteda`'s excellent [official tutorial](https://tutorials.quanteda.io) and [documentation](https://quanteda.io). 
+
```

Comment:
Are we linking to our repo somewhere here too?

Reply:
Yes, added to the introduction